msteams-platform/task-modules-and-cards/cards/cards-reference.md

```diff
@@ -126,8 +126,9 @@ The following table provides the features that support Adaptive Cards:
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for bot sent cards and action based message extensions.
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for other capabilities, such as cards sent by user (search based message extensions and link unfurling), tabs, and task modules.
 > * Positive or destructive action styling is not supported in Adaptive Cards on the Teams platform.
-> * Media elements are currently not supported in Adaptive Card on the Teams platform.
+> * Media elements are not supported in Adaptive Card on the Teams platform.
```

Comment:
We should remove this line. 

https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/media-elements-in-adaptive-cards?tabs=desktop

Reply:
Updated.